presto-kafka/src/test/java/io/prestosql/plugin/kafka/KafkaQueryRunner.java

```diff
@@ -99,6 +102,12 @@ public Builder setExtension(Module extension)
             return this;
         }
 
+        public Builder setPluginFactory(Function<Module, Plugin> pluginFactory)
```

Comment:
why this one is needed? Do we lack some extensions?

Reply:
Let me check it again.